src/components/Footer/Footer.js

```diff
@@ -1,7 +1,223 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
 
 const Footer = () => {
-  return <div></div>;
+  const [drop, setdrop] = useState(false);
+  const onClick = () => setdrop(!drop);
+
+  return (
+    <FooterBox>
+      <Header>
+        <Subject>
+          <ul>
+            <li>고객센터</li>
+            <li>공지사항</li>
+          </ul>
+        </Subject>
+        <Section>
+          <ul>
+            <li className="list">
+              <ul>
+                <li>
+                  <strong>RIDI</strong>PAPER
+                </li>
+                <li>제휴카드</li>
+                <li>뷰어 다운로드</li>
+                <li>리디캐시 충전</li>
+              </ul>
+            </li>
+            <li className="list">
+              <ul>
+                <li>콘텐츠 제공 문의</li>
+                <li>CP 사이트</li>
+                <li>사업 제휴 문의</li>
+                <li>페이퍼 대량구매 안내</li>
+              </ul>
+            </li>
+            <li className="list">
+              <ul>
+                <li>페이스북</li>
+                <li>인스타그램</li>
+              </ul>
+            </li>
+            <li className="list">
+              <ul>
+                <li>회사소개</li>
+                <li>인사채용</li>
+              </ul>
+            </li>
+          </ul>
+        </Section>
+      </Header>
+      <Company>
+        <div>리디(주) 사업자 정보</div>
+        <button onClick={onClick}>⌵</button>
+        {drop && <DropInfo />}
+        <DropInfo className="dropBox">
+          <div className="dropInfo">
+            <div>대표자&ensp;리디북슬북슬</div>
+            <div>사업자 등록번호&nbsp;123-45-67890</div>
+            <div>통신판매업 신고번호&ensp;제 2021-서울강남 12-34567호</div>
+          </div>
+          <div className="dropInfo">
+            <div>이메일&ensp;wecode@wecode.com</div>
+            <div>대표전화&ensp;1234-5678</div>
+            <div>
+              주소&ensp;서울시 강남구 테헤란로 427 위워크 선릉2호점 10층
+            </div>
+          </div>
+        </DropInfo>
```

Comment:
drop state에 따라서 조건부로 렌더링 하시려는 걸까요? 현재 drop state에 따라 렌더링 되는 것이 아니라 초기부터 렌더링 되고 있겠네요! 다음처럼 사용하셔야 할 것 같습니다.

```suggestion
        {drop && (
          <DropInfo className="dropBox">
              ...
          </DropInfo>
        )}
```

Reply:
네 맞습니다! 클릭하면 drop메뉴가 나오도록 하려고 하는데 말씀 주신 거처럼 적용하겠습니다.